pluto_build/sql/corr_ownername_punctuation.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE pluto a
```

Comment:
I suggest adding a comment up tod describing what this snippet of code does.  You can use other .sql scripts as examples.